velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -900,6 +900,11 @@ TEST_F(StringTest, translate) {
   testTranslate({"abc", std::nullopt, "\u2029"}, std::nullopt);
   testTranslate({"abc", "\u2028", std::nullopt}, std::nullopt);
   testTranslate({std::nullopt, "\u2028", "\u2029"}, std::nullopt);
+  // Special case
```

Comment:
Would you clarify why this case is special rather than just saying 'special case'?